tests/components/kraken/test_sensor.py

```diff
@@ -0,0 +1,89 @@
+"""Tests for the kraken sensor platform."""
+from datetime import timedelta
+from unittest.mock import patch
+
+from pykrakenapi.pykrakenapi import KrakenAPIError
+
+from homeassistant.components.kraken.const import DEFAULT_SCAN_INTERVAL, DOMAIN
+from homeassistant.const import EVENT_HOMEASSISTANT_START
+import homeassistant.util.dt as dt_util
+
+from .const import TICKER_INFORMATION_RESPONSE, TRADEABLE_ASSET_PAIR_RESPONSE
+
+from tests.common import async_fire_time_changed
+
+
+async def test_sensor(hass):
+    """Test that sensor has a value."""
+    utcnow = dt_util.utcnow()
+    # Patching 'utcnow' to gain more control over the timed update.
+    with patch("homeassistant.util.dt.utcnow", return_value=utcnow):
+        with patch(
+            "pykrakenapi.KrakenAPI.get_tradable_asset_pairs",
+            return_value=TRADEABLE_ASSET_PAIR_RESPONSE,
+        ):
+            with patch(
+                "pykrakenapi.KrakenAPI.get_ticker_information",
+                return_value=TICKER_INFORMATION_RESPONSE,
+            ):
+                result = await hass.config_entries.flow.async_init(
```

Comment:
We don't need to use the config flow to set up the integration. Just create a mock config entry and then set that up with `hass.config_entries.async_setup`.

Reply:
Done